diff --git a/cryptohome/mount_encrypted/encryption_key.cc b/cryptohome/mount_encrypted/encryption_key.cc
index feba33c6d8..2729c6f843 100644
--- a/cryptohome/mount_encrypted/encryption_key.cc
+++ b/cryptohome/mount_encrypted/encryption_key.cc
@@ -10,6 +10,7 @@
 #include <utility>
 
 #include <base/files/file_util.h>
+#include <base/strings/string_util.h>
 #include <base/strings/string_number_conversions.h>
 
 #include <brillo/file_utils.h>
@@ -210,7 +211,7 @@ result_code EncryptionKey::SetInsecureFallbackSystemKey() {
   std::string product_uuid;
   if (base::ReadFileToStringWithMaxSize(base::FilePath(paths::kProductUUID),
                                         &product_uuid, kMaxReadSize)) {
-    system_key_ = Sha256(product_uuid);
+    system_key_ = Sha256(base::ToUpperASCII(product_uuid));
     LOG(INFO) << "Using UUID as system key.";
     system_key_status_ = SystemKeyStatus::kProductUUID;
     return RESULT_SUCCESS;
