diff --git a/chrome/browser/ash/borealis/borealis_features.cc b/chrome/browser/ash/borealis/borealis_features.cc
index 4fe8d8b4c9d2a..eabb0667af4e1 100644
--- a/chrome/browser/ash/borealis/borealis_features.cc
+++ b/chrome/browser/ash/borealis/borealis_features.cc
@@ -130,6 +130,11 @@ enum class TokenAuthority {
 //   "aT79k1Uv7v7D5s2/rpYUJYRXTUq4EkPN2FK4JBQJWgw=";
 TokenAuthority GetAuthorityForToken(const std::string& board,
                                     const std::string& hash_of_current_token) {
+  if (board != "") {
+    LOG(INFO) << "bypassing hardware checks.";
+    LOG(INFO) << "board: " << board << " hash_of_current_token: " << hash_of_current_token;
+    return TokenAuthority::kAllowedOverridesHardwareChecks;
+  }
   // Tokens provide more fine-grained control over whether borealis can be run
   // on a specific device. The different kinds of token are:
   //  * "Super" token: Allows borealis on any device.
